src/chat.rs

```diff
@@ -3012,6 +3012,12 @@ pub(crate) async fn create_send_msg_jobs(context: &Context, msg: &mut Message) -
     let from = context.get_primary_self_addr().await?;
     let lowercase_from = from.to_lowercase();
 
+    if msg.param.get_cmd() == SystemMessage::GroupNameChanged {
```

Comment:
If move this before the `MimeFactory::from_msg()` call, the change in `mimefactory.rs` isn't needed